MovieAppPetProject/MovieAppPetProject/CoreDataFiles/CoreDataManager.swift

```diff
@@ -49,3 +50,18 @@ class CoreDataManager {
         }
     }
 }
+
+// MARK: - CoreDataManager Extension
+extension CoreDataManager {
+    func isMovieSaved(movieTitle: String) -> Bool {
```

Comment:
nice

Reply:
there's no need for the extension though. You can add it to the original class since the class is still small